dev/ScrollViewer/ScrollViewer.cpp

```diff
@@ -529,7 +536,12 @@ void ScrollViewer::OnScrollViewerPointerEntered(
     {
         // Mouse/Pen inputs dominate. If touch panning indicators are shown, switch to mouse indicators.
         m_preferMouseIndicators = true;
-        ShowIndicators();
+
+        UpdateScrollControllersAutoHiding();
+        if (AreScrollControllersAutoHiding())
+        {
+            UpdateVisualStates(true /*useTransitions*/, true /*showIndicators*/);
+        }
```

Comment:
These 4 lines of code show up quite a bit in the change, consider refactoring into a helper.

Reply:
I ended up adding two params to UpdateVisualStates instead: bool updateScrollControllersAutoHiding = false & bool onlyForAutoHidingScrollControllers = false.